commit 0a3f87bd2a52afa2e002da8c1b0284cd948a555e
Author: Jason Gladish <jason@expectedbehavior.com>
Date:   Mon May 29 17:34:46 2023 -0400

    Fix issue with passing PrinceOptions
    
    Why is this change needed?
    --------------------------
    Without this the following code will not work:
    
    ```csharp
        doc = new Doc(
          name: "csharp-sync.pdf",
          test: true,
          documentUrl: "https://expired.badssl.com/",
          documentType: Doc.DocumentTypeEnum.Pdf,
          princeOptions: new PrinceOptions(
            insecure: true
          )
        );
    ```
    and will result in the following error (assuming `new PrinceOptions` is
    on line 43):
    ```
    test/prince_options.cs(43,22): error CS1729: The type `DocRaptor.Model.PrinceOptions' does not contain a constructor that takes `24' arguments
    /app/bin/DocRaptorWithDependencies.dll (Location of the symbol related to previous error)
    Compilation failed: 1 error(s), 0 warnings
    ```
    
    How does it address the issue?
    ------------------------------
    Switching the argument default to `null` prevents the error. This should
    be fine because the constructor checks if this argument is null and
    defaults to `InputEnum.Html` if it is.
    
    It's not clear to me why this errors in the first place, it seems like
    using an enum as a default argument should be fine, but I'm not super
    familiar with C#.
    
    Did you complete all of the following?
    --------------------------------------
    - Run test suite?
    - Add new tests?
    - Consider security implications and practices?

diff --git a/src/DocRaptor/Model/PrinceOptions.cs b/src/DocRaptor/Model/PrinceOptions.cs
index e60b6fb..20a42ef 100644
--- a/src/DocRaptor/Model/PrinceOptions.cs
+++ b/src/DocRaptor/Model/PrinceOptions.cs
@@ -96,7 +96,7 @@ namespace DocRaptor.Model
         /// <param name="cssDpi">Set the DPI when rendering CSS. Defaults to 96 but can be set with Prince 9.0 and up..</param>
         /// <param name="profile">In Prince 9.0 and up you can set the PDF profile..</param>
         /// <param name="pdfTitle">Specify the PDF title, part of the document&#39;s metadata..</param>
-        public PrinceOptions(string baseurl = default(string), bool noXinclude = default(bool), bool noNetwork = default(bool), bool noParallelDownloads = default(bool), string httpUser = default(string), string httpPassword = default(string), string httpProxy = default(string), int httpTimeout = default(int), bool insecure = default(bool), string media = "print", bool noAuthorStyle = default(bool), bool noDefaultStyle = default(bool), bool noEmbedFonts = default(bool), bool noSubsetFonts = default(bool), bool noCompress = default(bool), bool encrypt = default(bool), int keyBits = default(int), string userPassword = default(string), string ownerPassword = default(string), bool disallowPrint = default(bool), bool disallowCopy = default(bool), bool disallowAnnotate = default(bool), bool disallowModify = default(bool), bool debug = default(bool), InputEnum? input = InputEnum.Html, string version = default(string), bool javascript = default(bool), int cssDpi = default(int), string profile = default(string), string pdfTitle = default(string))
+        public PrinceOptions(string baseurl = default(string), bool noXinclude = default(bool), bool noNetwork = default(bool), bool noParallelDownloads = default(bool), string httpUser = default(string), string httpPassword = default(string), string httpProxy = default(string), int httpTimeout = default(int), bool insecure = default(bool), string media = "print", bool noAuthorStyle = default(bool), bool noDefaultStyle = default(bool), bool noEmbedFonts = default(bool), bool noSubsetFonts = default(bool), bool noCompress = default(bool), bool encrypt = default(bool), int keyBits = default(int), string userPassword = default(string), string ownerPassword = default(string), bool disallowPrint = default(bool), bool disallowCopy = default(bool), bool disallowAnnotate = default(bool), bool disallowModify = default(bool), bool debug = default(bool), InputEnum? input = null, string version = default(string), bool javascript = default(bool), int cssDpi = default(int), string profile = default(string), string pdfTitle = default(string))
         {
             this.Baseurl = baseurl;
             this.NoXinclude = noXinclude;
